src/Engine/Graphics/Nuklear.h

```diff
@@ -36,16 +42,24 @@ class Nuklear {
      bool Reload();
      void Release(WindowType winType);
      void Destroy();
+     void addInitLuaLibs(std::function<void(lua_State *)> callback);
+     void addInitLuaFile(const char *lua_file);
+     bool isInitialized(WindowType winType) const;
      enum NUKLEAR_MODE Mode(WindowType winType);
 
      struct nk_context *ctx = nullptr;
 
+     static std::unique_ptr<LogSink> createNuklearLogSink();
+
  private:
      void Release(WindowType winType, bool reload);
      bool LuaInit();
      void LuaRelease();
      bool LuaLoadTemplate(WindowType winType);
 
+     std::vector<std::string> _init_lua_files;
+     std::vector<std::function<void(lua_State *)>> _init_lua_lib_callbacks;
```

Comment:
Codestyle: `_initLuaLibCallbacks`

Reply:
whoops. that's the style I followed on the lua scripts... they bled here as well! 😅